Documentation/guides/LiveCode Script.md

```diff
@@ -294,9 +315,13 @@ of links to the most recent posts which have been made.
 
 ##### Blog
 
+This button takes you to LiveCode's online blog, where LiveCode engineers and developers regularly post updates about what's new in LiveCode, how-to tutorials, and announcements of interest to the LiveCode developer community.
```

Comment:
@asayd This and the hunk below should probably be wrapped otherwise this seems good to me